src/lib/Theme.ts

```diff
@@ -1112,8 +1112,37 @@ export default class Theme {
             throw new CommandError(err.message, err.code);
         }
     };
+    
+    public static generateAssets = async () =>{
+        let available_sections = await Theme.getAvailableSectionsForSync();
+        await Theme.validateAvailableSections(available_sections);
+        Theme.clearPreviousBuild();
```

Comment:
Move this on first line in `generateAssets` function.

Reply:
Already added inside theme package cmd before calling generateAssets